schemas/teacher.js

```diff
@@ -4,15 +4,22 @@ const { Schema } = mongoose;
 
 const teacherSchema = Schema({
   name: { type: String, required: true },
-  phone: { type: String, required: true },
```

Comment:
I removed phone from the model because that wasn't included in the sample data provided by Nathan.  Is there a reason we should be keeping it in? @warlordlizard

Reply:
I would keep `phone` in the model, but remove the `required: true` qualifier. Nathan may have redacted real phone numbers from the sample data, and either way it doesn't hurt to have it there as an optional field.